src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    stdPublish: 'stdPublish',
+    stdscreenshare: 'Deseja compartilhar sua tela?',
+    studentAudEnable: 'Áudio do estudante habilitado',
+    studentSafariBrowserIssue: 'Seu browser {virtualclass1} {virtualclass2} não pode compartilhar sua Webcam com outros usuários. Congrea suporta completamente Chrome e Firefox.',
+    supportDesktop: 'Sala Virtual não suporta tablets nem smartphones. Para apresentar, favor usar um computador.',
+    supportDesktopOnly: 'Desculpe, o modo de apresentador só suporta computadores. Tablets e smartphones não são suportados.',
+    teacherSafariBrowserIssue: 'Safari não suporta a funcionalidade de apresentador. Favor trocar para Chrome ou Firefox.',
+    techsupport: 'Suporte técnico',
+    testingbrowser: 'Testando compatibilidade com o browser',
+    testinginternetspeed: 'Testando velocidade da internet',
+    testingmichrophone: 'Testando microfone',
+    testingspeaker: 'Testando caixa de som',
+    testingwebcam: 'Testando Webcam',
+    Text: 'Texto',
+    Text_Limit_Exeed: 'Alerta: Alto uso TX',
+    TextEditor: 'Editor de texto',
+    textPlaceholder: 'Inserir texto aqui',
+    Time: 'Tempo',
+    Tmyclose: 'Professor pode fechar esta enquete a qualquer momento',
+    total: 'Total',
+    totalprogress: 'Progresso total',
+    tpAudioTest: 'Testar áudio',
+    transferControls: 'Transferir controles',
+    Triangle: 'Triângulo',
+    Udocument: 'Upload de documento',
+    Unauthenticated: 'Erro: Acesso inválido',
+    unmuteAll: 'Desmutar todos',
+    uploadsession: 'Favor aguarde até o processamento acabar',
+    uploadvideo: 'Upload de vídeo',
+    uploadedsession: 'Sua sessão se encerrou. Você já pode fechar sua janela '
+    + ' ou fechar este popup para iniciar uma nova sessão.',
+    usermissing: 'Usuário está faltando',
+    userList: "Lista de usuários",
+    upvote: 'Votar a favor',
+    instructorVideo: 'Vídeo do professor',
+    validateurlmsg: 'URL inválida',
+    VCE2: 'Sem permissão para gravar sessão',
+    VCE4: 'Dados de registro estão faltando',
+    VCE5: 'Incapaz de Registrar Dados.',
+    VCE6: 'Módulo de curso está faltando.',
+    VideodbHeading: 'Painel de vídeo',
+    videoInput: 'Entrada de vídeo',
+    videooff: 'Vídeo desligado',
+    videoon: 'Vídeo ligado',
+    videoquality: 'Qualidade do vídeo',
+    virtualclassnoteHeader: 'Adicionar Anotação',
+    VotedSoFar: 'Votos recebidos até agora',
+    votesuccess: 'Seu voto foi enviado com sucesso. O resultado será mostrado em breve.',
+    votesuccessPbt: 'Voto enviado com sucesso. Você não tem permissão para ver o resultado.',
+    Vwllbshrshortly: 'Vídeo será compartilhado em breve',
+    viewall: 'Ver tudo',
+    waitmsgconnect: 'Favor aguarde um pouco. A aplicação está tentando se conectar.',
+    watstdrespo: 'Esperando a resposta do estudante',
+    wbrtcMsgChrome: 'Você pode clicar no botão de negação para não compartilhar seu microfone e câmera com o Congrea.' + 'ou clicar no botão de permitir para compartilhá-los.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 176. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)